x-pack/plugins/security_solution/public/common/components/event_details/table/use_action_cell_data_provider.ts

```diff
@@ -72,6 +74,7 @@ export const useActionCellDataProvider = ({
   isObjectArray,
   linkValue,
   values,
+  alertsOnly = true,
```

Comment:
Since this has the side effect of adding an additional filter, how do you feel about making `alertsOnly` non-optional, which would require all consumers to explicitly opt-in to this behavior?